lldb/test/API/lang/swift/async/formatters/task/main.swift

```diff
@@ -0,0 +1,10 @@
+func f() {
+    let task = Task {
+        // Extend the task's lifetime, hopefully long enough for the breakpoint to hit.
```

Comment:
Could the task wait for a lock that is released on line 8, so we don't need to rely on the sleep?

Reply:
the reason I didn't use a lock because I wasn't sure if there were portable locks. The sleep isn't clean but should be long enough to get the job done. But if you know of a portable lock I can use that.

Reply:
For a test, you could also just make it loop and sleep for a long time. It won’t prevent the program from exiting.

Reply:
The asanified LLDB often has random delays in the multiple seconds range but ew also can't just sleep for 5s without making the test suite run 5s longer.
Actually — could you sleep for 60s and do a process.Kill() in the python test code?

Reply:
The sleep is in an unstructured background task so it won’t hold up the rest of the program.

Reply:
> For a test, you could also just make it loop and sleep for a long time.

why a loop vs a single long sleep?